jwst/photom/photom.py

```diff
@@ -903,9 +903,13 @@ def photom_io(self, tabdata, order=None):
                 if unit_is_surface_brightness:
                     slit.meta.bunit_data = 'MJy/sr'
                     slit.meta.bunit_err = 'MJy/sr'
+                    self.input.meta.bunit_data = 'MJy/sr'
```

Comment:
Given that this is within some much larger loop over all the slits within a `MultiSlitModel` the values in the main header will get multiple updates, which isn't necessary (one is sufficient), but it's also not a big enough deal to worry about changing anything.

Reply:
@hbushouse 
A better solution is the one you suggested and setting self.input.meta_bunit_data = None. 
This removes the bunit keyword from top header.